meinberlin/assets/scss/components/_action.scss

```diff
@@ -7,10 +7,27 @@
     font-size: $font-size * 100%;
 }
 
+.actions {
+    padding: 0;
+    border: 1px solid $border-color;
+
+    @media (min-width: $breakpoint) {
+        padding: 1em;
+    }
+
+    h2 {
+        margin-top: 0;
+    }
+}
```

Comment:
I don't like having two components on a single DOM element. Can we change this to `.block--bordered`?